test/vectors.test.ts

```diff
@@ -1,14 +1,23 @@
 /* eslint-disable @typescript-eslint/restrict-template-expressions */
-
 import { hexToBytes } from '@metamask/utils';
 
 import type { SLIP10PathTuple } from '../src';
 import { secp256k1 } from '../src';
 import type { Curve } from '../src/curves';
-import { ed25519 } from '../src/curves';
-import { createBip39KeyFromSeed } from '../src/derivers/bip39';
+import { ed25519Bip32, ed25519 } from '../src/curves';
+import {
+  createBip39KeyFromSeed,
+  entropyToCip3IcarusRootKeyPair,
```

Comment:
naming nitpick: `keypair` is a single word actually so I'd consider changing the name to 
```suggestion
  entropyToCip3IcarusRootKeypair,
```

Reply:
ok I checked github for the more prevalent casing of this word and it seems 50/50, so I guess let's leave it like it is 🤷 

https://github.com/search?q=keypair&type=code&p=2